core/include/kernel/dt.h

```diff
@@ -83,6 +83,7 @@ struct dt_descriptor {
 #ifdef _CFG_USE_DTB_OVERLAY
 	int frag_id;
 #endif
+	struct transfer_list_header *tl;
```

Comment:
I would prefer if we didn't add this. Is it possible to do without it?

Reply:
Fixed.